libs/src/sdk/types/types.ts

```diff
@@ -0,0 +1,179 @@
+// TODO: split up this file
+
+export type Maybe<T> = T | undefined
+export type Nullable<T> = T | null
+
+type CID = string
+type ID = number
+type UID = string
+
+export type UserMetadata = {
+  user_id: number
+  album_count: number
+  artist_pick_track_id: Nullable<number>
+  bio: string | null
+  cover_photo: Nullable<CID>
+  creator_node_endpoint: string
+  current_user_followee_follow_count: number
+  does_current_user_follow: boolean
+  followee_count: number
+  follower_count: number
+  supporter_count: number
+  supporting_count: number
+  handle: string
+  handle_lc: string
+  is_deactivated: boolean
+  is_verified: boolean
+  is_storage_v2: boolean
+  location: Nullable<string>
+  // this should be removed
```

Comment:
JC, When/how can we remove it?

Reply:
🤷 I just copied this from libs

I do think `is_creator` can theoretically be removed now though